src/pages/students/forms/studentDataForm/StudentData.tsx

```diff
@@ -36,64 +47,58 @@ function StudentData() {
     resolver: yupResolver(StudentDataScheme),
     defaultValues: {
     ...studentRegistrationForm, 
-    birthday: studentRegistrationForm.birthday
-      ? new Date(studentRegistrationForm.birthday).toISOString().split('T')[0]
-      : "", 
   }
   });
 
   const navigate = useNavigate();
   
   const onSubmit = (studentData: FormData) => {
-    const normalizedStudentData = {
-      ...studentData,
-      pathologies: studentData.pathologies ?? undefined,
-       birthday: studentData.birthday
-      ? new Date(studentData.birthday).toISOString()
-      : undefined,
-    };
-    dispatch(setStudentData(normalizedStudentData)),
-    navigate("/datos-del-pago");
+    createStudentCall({
+      name: studentData.name,
+      lastName: studentData.lastName,
+      dni: studentData.dni,
+      cellphoneNumber: studentData.cellphoneNumber,
+      birthday: studentData.birthday,
+      pathologies: studentData.pathologies
+    })
```

Comment:
Deberias poder hacer lo siguiente tambien


```suggestion
    createStudentCall({...studentData})
```